core/src/main/scala/org/apache/spark/api/java/JavaPairRDD.scala

```diff
@@ -233,18 +235,44 @@ class JavaPairRDD[K, V](val rdd: RDD[(K, V)])
   def combineByKey[C](createCombiner: JFunction[V, C],
     mergeValue: JFunction2[C, V, C],
     mergeCombiners: JFunction2[C, C, C],
-    partitioner: Partitioner): JavaPairRDD[K, C] = {
+    partitioner: Partitioner,
+    mapSideCombine: Boolean,
+    serializer: Serializer): JavaPairRDD[K, C] = {
```

Comment:
It's weird because we don't specify the serializer in the docs for PairRDDFunctions.